.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,19 @@
+name: Nightly
+on:
+  schedule:
+    - cron: 0 8 * * 1-5
+  workflow_dispatch:
+
+jobs:
+  trunk_check:
+    name: Trunk Check Upload
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Trunk Check
+        uses: trunk-io/trunk-action@v1.0.0
+        with:
+          trunk-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This should be a token you get from the trunk web app and store as a GH action secret, not the GitHub token

Reply:
@dapirian Ooo! That wasn't clear to me from the instructions. I don't see any part of the Trunk web app where I can get this from. Maybe related to the GitHub connection issue you're looking into though?

Reply:
oh we'll make that more clear! yes, you can't see it because of that issue - stay tuned :)